src/commons/application/ApplicationTypes.ts

```diff
@@ -207,26 +209,29 @@ export const styliseSublanguage = (chapter: Chapter, variant: Variant = Variant.
   }
 };
 
-export const sourceSublanguages: Language[] = [
-  { chapter: Chapter.SOURCE_1, variant: Variant.DEFAULT },
+export const sourceSublanguages = [
+  { chapter: Chapter.SOURCE_1, variant: Variant.DEFAULT, usesSubstVisualizer: true },
   { chapter: Chapter.SOURCE_1, variant: Variant.TYPED },
   { chapter: Chapter.SOURCE_1, variant: Variant.WASM },
   { chapter: Chapter.SOURCE_1, variant: Variant.LAZY },
-  { chapter: Chapter.SOURCE_1, variant: Variant.NATIVE },
-  { chapter: Chapter.SOURCE_2, variant: Variant.DEFAULT },
+  { chapter: Chapter.SOURCE_1, variant: Variant.NATIVE, usesSubstVisualizer: true },
+
```

Comment:
I'm surprised the linter doesn't complain!

Reply:
I believe if I try to add another key, it'll auto convert to having each key in a separate line. This happens to be just nicely within the line length limit :)